packages/create-strapi-starter/utils/fetch-github.js

```diff
@@ -45,32 +47,38 @@ async function getDefaultBranch(repo) {
  */
 async function getRepoInfo(starter) {
   const repoInfo = await parseGitUrl(starter);
-  const { name, full_name, ref, protocols, source } = repoInfo;
+  const { name, full_name, ref, filepath, protocols, source } = repoInfo;
 
   if (protocols.length === 0) {
     return getShortcut(starter);
   }
 
   if (source !== 'github.com') {
-    stopProcess(`Github URL not found for: ${chalk.yellow(starter)}`);
+    stopProcess(`GitHub URL not found for: ${chalk.yellow(starter)}.`);
   }
 
   return {
     name,
     full_name,
     ref,
+    filepath,
   };
 }
 
 /**
  * @param  {string} starterUrl Github url for strapi starter
  * @param  {string} tmpDir Path to temporary directory
  */
-async function downloadGithubRepo(starterUrl, tmpDir) {
-  const { full_name, ref, usedShortcut } = await getRepoInfo(starterUrl);
+async function downloadGitHubRepo(starterUrl, tmpDir) {
+  const { full_name, ref, filepath, usedShortcut } = await getRepoInfo(starterUrl);
   const default_branch = await getDefaultBranch(full_name);
 
-  const branch = ref ? ref : default_branch;
+  let branch = default_branch;
```

Comment:
if we return branch from getRepoInfo this makes this code way simpler

Reply:
Done in both packages :)
I took the opportunity to change how we parse the shorthands a bit to unify the return values, I hope that's ok